src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -184,17 +184,12 @@ uct_cuda_ipc_mem_add_reg(void *addr, uct_cuda_ipc_memh_t *memh,
             UCT_CUDADRV_FUNC_LOG_ERR(cuMemExportToShareableHandle(
                         &key->ph.handle.fabric_handle, handle,
                         CU_MEM_HANDLE_TYPE_FABRIC, 0));
+        UCT_CUDADRV_FUNC_LOG_WARN(cuMemRelease(handle));
```

Comment:
it changes the behavior that now we do not fail if `cuMemRelease` fails

Reply:
Yes, this is intentional. This is an auxiliary resource that is allocated on line 180. According to the existing logic, we print a warning message and continue working if the resource release fails.